Godeps/_workspace/src/bazil.org/fuse/syscallx/syscallx_std.go

```diff
@@ -3,24 +3,24 @@
 package syscallx
 
 // This file just contains wrappers for platforms that already have
-// the right stuff in stdlib.
+// the right stuff in golang.org/x/sys/unix.
 
 import (
-	"syscall"
+	"golang.org/x/sys/unix"
```

Comment:
I'm even more confused now..

I cloned this branch into an empty GOPATH on two non-darwin machines and it builds fine..
Am I missing something about this `syscallx` package?

cc @tv42